src/main/java/User_Register_System/UserRegUITemplate.java

```diff
@@ -0,0 +1,214 @@
+package User_Register_System;
+
+import data_access_storage.UserRepoManager;
+import entities.*;
+
+import javax.swing.*;
+import java.awt.event.ActionListener;
+import java.awt.event.ActionEvent;
+
+
+public class UserRegUITemplate extends JPanel{
+    UserRegUIController userRegUIController;
+
+    /**
+     * This method will add space (" ") when the length is too short.
+     * */
+    public static String padLeft(String origin, int length, char ch) {
+        while (origin.length() < length) {
+            origin = ch + origin;
+        }
+        return origin;
+    }
+    public UserRegUITemplate(UserRepoManager users,UserRegUIController userRegUIController){
```

Comment:
If you dont need UserRepoManager users, please remove this parameter from your constructor

Reply:
Removed UserRepoManager, tests will update shortly.